cmake/external/onnxruntime_external_deps.cmake

```diff
@@ -100,6 +100,44 @@ FetchContent_Declare(
     FIND_PACKAGE_ARGS 1.12.0...<2.0.0 NAMES Flatbuffers
 )
 
+# Download a protoc binary from Internet if needed
+if(CMAKE_CROSSCOMPILING AND NOT ONNX_CUSTOM_PROTOC_EXECUTABLE AND NOT CMAKE_OSX_ARCHITECTURES)
```

Comment:
what does macOS users do? are they required to use build with '--path_to_protoc_exe' flag?

Reply:
No. It should work directly.

Reply:
See this: https://github.com/microsoft/onnxruntime/blob/main/cmake/external/onnxruntime_external_deps.cmake#L318